### DISCUSSION TITLE
p/moving-away-from-github-pages-to-cloudflare-pages/

### DISCUSSION THREAD

**Discussion by giscus** [donhector/donhector.github.io#24] [Giscus]
# p/moving-away-from-github-pages-to-cloudflare-pages/

Some of the reasons why I moved

https://hmolina.dev/p/moving-away-from-github-pages-to-cloudflare-pages/

**Comment by eli32-vlc** [comment]
Amazing Post!!

**Reply by donhector** [reply]
Glad you liked it!
